### ISSUE TITLE
Train/Test Split of CUBES

### ISSUE THREAD

**Issue by vinitveersingh** [ranahanocka/MeshCNN#120]
Hello,
Hope you are well.
I downloaded the CUBES data set from the provided link and noticed the train/test split is different from the one mentioned in the original paper. In the original paper, the train/test is mentioned as 3910/690 with 23 classes. However, the downloaded link provides a train/test split of 3722/659 with 22 classes. Also upon running the training script the best accuracy for 600 epochs comes out to be 96.66 compared to 92.16 reported in the paper.

I would really appreciate some clarification on this.

Thank you
